docs/langref.md

```diff
@@ -768,33 +769,9 @@ Example := Rectangle {
 }
 ```
 
-It is possible to re-expose a callback or properties from a global using the two way binding syntax.
```

Comment:
I think this should stay in this reference.

> While that still works of course, we don't need to mention it as the first example.

But this page is a reference and should document the whole language entirely.

I guess now this page serves both as a reference and as a introduction to the language and that's maybe a mistake.
Maybe the other reciepes you are working on vould be used as an introduction to the language instead.

Reply:
Maybe not quite "introduction" but any example we provide (including in which order) becomes a "best practice".

If you'd like to retain this in the reference, then I can add a sentence to the two-way bindings instead?